### ISSUE TITLE
Translate all logger events to Lager events

### ISSUE THREAD

**Issue by hauleth** [erlang-lager/lager#521]
This is similar approach that Elixir's logger take some time ago, where Lager messages still will be handled by the Lager only, but it will also get messages from the `logger` module. This should easy the migration like mentioned in #519 where all messages are passed through `logger` from day 1.

**Comment by mrallen1** [comment]
I think @Vagabond is planning along these lines for the future 4.x version of lager. Thanks for the issue though.

**Comment by hauleth** [comment]
Yes, I have seen their work. My point is to reduce scope for a moment, and do not rewrite all log commands as well, but for now just provide logging facilities for `logger` (and soon Elixir's `Logger`) users.

**Comment by g-andrade** [comment]
> My point is to reduce scope for a moment, and do not rewrite all log commands as well, [...]

In case more elegant alternatives don't come up within a short time frame, [fake-lager](https://github.com/g-andrade/fake-lager) might be useful to you, but it's still rather experimental.
